opts/opts.go

```diff
@@ -515,3 +516,26 @@ func (m *MemSwapBytes) UnmarshalJSON(s []byte) error {
 	b := MemBytes(*m)
 	return b.UnmarshalJSON(s)
 }
+
+type OomScoreAdj int64
+
+func (o *OomScoreAdj) Type() string { return "int64" }
+
+func (o *OomScoreAdj) Value() int64 { return int64(*o) }
+
+func (o *OomScoreAdj) String() string {
+
+	val := strconv.FormatInt(int64(*o), 10)
+	return val
+}
+
+func (o *OomScoreAdj) Set(value string) error {
+
+	var conv int64
+	conv, err := strconv.ParseInt(value, 10, 64)
+	if err != nil || conv < -1000 || conv > 1000 {
+		return err
+	}
```

Comment:
In the case where `err == nil` but `conv` is out of range, [the `Set()` call will silently be a no-op.](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) Either generate a client-side error, or (preferably) do what @thaJeztah recommends and leave range validation to the API / daemon.

Reply:
Yes, I think we can keep it simple for now, and do the same as we do for `docker run`, which is a regular `IntVar`.

I tend to reduce client-side validation to a minimum to keep the CLI agnostic and make the daemon "source of truth". That's not all set in stone, and there are some grey areas where it's worth the trade-ff but I'd like to avoid the CLI making assumptions that may not be correct.